### PR TITLE
Configuring Windows Auth

### LINKED ISSUE #2711: Document how to setup windows authentication in ASP.NET Core
I had someone point out to me today that we don't currently cover setting up Windows authentication anywhere in our docs.

[Edit Rick] [@]Tratcher  is contact when ready to start to review outline.

### PR DESCRIPTION
**Pull request by ardalis**
[Review URL](https://review.docs.microsoft.com/en-us/aspnet/core/security/authentication/windowsauth?branch=pr-en-us-3622)
Fixes #2711

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@ardalis we don't need a download sample for this. Delete all the files that don't contain snippets.

**Comment by ardalis** [comment]
Looks like that's the whole sample at this point.

File: aspnetcore/security/authentication/windowsauth.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+## What is Windows authentication
+
+Windows authentication relies on the underlying operating system to authenticate users of ASP.NET Core apps. For obvious reasons, it only works on apps running on Windows hosts. You can use Windows authentication when your server runs on a corporate network using Active Directory domain identities or other Windows accounts to identify users. Windows authentication is a secure form of authentication best suited to intranet environments where client applications and web servers belong to the same Windows domain.
+
+[Learn more about Windows Authentication](https://www.iis.net/configreference/system.webserver/security/authentication/windowsauthentication).
+
+## Enabling Windows authentication in your app
+
+You can use Window authentication in your ASP.NET Core app hosted in IIS Express by installing the appropriate template or configuring the app's properties.
```
Comment (Tratcher) [inline aspnetcore/security/authentication/windowsauth.md]: installing? There's no additional install for the template.

File: aspnetcore/security/authentication/windowsauth.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+By [Steve Smith](https://ardalis.com)
+
+Windows authentication can be configured for ASP.NET Core apps hosted with IIS or WebListener.
+
+## What is Windows authentication
+
+Windows authentication relies on the underlying operating system to authenticate users of ASP.NET Core apps. For obvious reasons, it only works on apps running on Windows hosts. You can use Windows authentication when your server runs on a corporate network using Active Directory domain identities or other Windows accounts to identify users. Windows authentication is a secure form of authentication best suited to intranet environments where client applications and web servers belong to the same Windows domain.
+
+[Learn more about Windows Authentication](https://www.iis.net/configreference/system.webserver/security/authentication/windowsauthentication).
```
Comment (Tratcher) [inline aspnetcore/security/authentication/windowsauth.md]: and installing it for IIS.

File: aspnetcore/security/authentication/windowsauth.md (hunk 1 above)
Comment (ardalis) [inline aspnetcore/security/authentication/windowsauth.md]: "choosing"

**Comment by Rick-Anderson** [comment]
@ardalis  can you put a boarder around  aspnetcore/security/authentication/windowsauth/_static/browser-screenshot.png ? Do that, merge/clean up my PR and then you can merge.

**Comment by ardalis** [comment]
I already merged your PR in. I'll add the border.

**Pull request merged**